test/pulp_api_test.rb

```diff
@@ -36,4 +36,28 @@ def test_returns_50X_on_socket_error
   ensure
     Net::HTTP.any_instance.unstub(:request)
   end
-end
\ No newline at end of file
+
+  def test_returns_pulp_disk_on_200
+    PulpProxy::Plugin.load_test_settings(:pulp_dir => ::Sinatra::Application.settings.root)
```

Comment:
You'll need to define 'pulp_content_dir' and 'mongodb_dir' too, as otherwise default values are going to be used (and test will fail on my machine, as I don't have those dirs).